### DISCUSSION TITLE
Svelte page changes not reflecting

### DISCUSSION THREAD

**Discussion by normanmadrid** [pocketbase/pocketbase#1574] [Q&A] [answered]
Sorry I'm new to extending a framework like this one, but when I make a change on a svelte page located in ui/src/components i dont see it reflect when I run the main.go serve command, some directions on how to properly rebuild everything would be great!

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure that I understand what you are trying to do.

If you want to use PocketBase as framework, please follow the docs under the "Advanced" section in https://pocketbase.io/docs/use-as-framework/.

The Admin UI at the moment cannot be extended without modyfing the source. If you want to maintain your own fork/custom PocketBase version and do changes in the UI, you can check the [ui/README](https://github.com/pocketbase/pocketbase/tree/master/ui) OR the instructions in https://github.com/pocketbase/pocketbase/blob/master/CONTRIBUTING.md#making-changes-in-the-admin-ui

**Reply by normanmadrid** [reply]
this answers my question, thanks alot!
